exercises/series/src/test/java/SeriesTest.java

```diff
@@ -63,11 +64,11 @@ public void keepsArbitraryDigitOrder() {
     public void canSliceByOne() {
         Series series = new Series("01234");
         List<List<Integer>> expected = Arrays.asList(
-                Arrays.asList(0),
-                Arrays.asList(1),
-                Arrays.asList(2),
-                Arrays.asList(3),
-                Arrays.asList(4)
+		Collections.singletonList(0),
```

Comment:
Spacing looks off by a single char in this line only, otherwise this PR looks great!